### PR TITLE
clean meson build system & add i18n

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
This PR does nothing more just generating the pot file for the current destkop & appdata file. 
and cleans the meson.build files a bit (using 2 spaces as an indent size, seems to be common by a lot of meson users)

### CODE REVIEW

**Review by danirabbit — CHANGES_REQUESTED** [review]
According to this diff you've added two new files instead of renaming old files. In the future you can use `git mv` to avoid this issue

Please don't change the indentation level of the meson files.

**Comment by bilelmoussaoui** [comment]
I've reverted the indent size to 4. Sorry for that, i always forget to use `git mv`

File: meson.build [hunk 1]
```diff
@@ -1,12 +1,15 @@
-project('com.github.danrabbit.harvey', ['vala', 'c'],
+project(
+    'com.github.danrabbit.harvey',
+    ['vala', 'c'],
```
Comment (danirabbit) [inline meson.build]: This should be in a different PR. These code style changes are not related to i18n changes

File: meson.build [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
     version: '0.1.4'
 )
 
 gnome = import('gnome')
 i18n = import('i18n')
 
 asresources = gnome.compile_resources(
-    'as-resources', 'data/css.gresource.xml',
+    'as-resources',
+    'data/css.gresource.xml',
```
Comment (danirabbit) [inline meson.build]: This is intentionally inline

File: meson.build [hunk 3]
```diff
@@ -15,11 +18,14 @@ add_project_arguments(['--vapidir', join_paths(meson.current_source_dir(), 'vapi
 
 conf = configuration_data()
 conf.set_quoted('GETTEXT_PACKAGE', meson.project_name())
-configure_file(output: 'config.h', configuration: conf)
+configure_file(
+    output: 'config.h',
+    configuration: conf
+)
```
Comment (danirabbit) [inline meson.build]: Same here. This code style change should be in a code style PR. Not related to i18n changes

File: meson.build [hunk 4]
```diff
@@ -15,11 +18,14 @@ add_project_arguments(['--vapidir', join_paths(meson.current_source_dir(), 'vapi
...
 config_h_dir = include_directories('.')
 
 c_args = [
-  '-include', 'config.h'
+    '-include', 'config.h'
```
Comment (danirabbit) [inline meson.build]: And this

File: meson/post_install.py [hunk 5]
```diff
@@ -1,10 +1,14 @@
...
+ICONS_DIR = path.join(PREFIX, 'share', 'icons', 'hicolor')
 
-if not os.environ.get('DESTDIR'):
-	print('Compiling gsettings schemas...')
-	subprocess.call(['glib-compile-schemas', schemadir])
+if not environ.get('DESTDIR'):
+    print('Compiling gsettings schemas...')
+    call(['glib-compile-schemas', SCHEMA_DIR])
+    print('Refreshing icon cache...')
+    call(['gtk-update-icon-cache', '-qtf', ICONS_DIR])
```
Comment (danirabbit) [inline meson/post_install.py]: This is also not related to i18n and should be in a separate PR

**Review by danirabbit — CHANGES_REQUESTED** [review]

**Comment by bilelmoussaoui** [comment]
Done

**Comment by danirabbit** [comment]
Can you fix the two files which have been added instead of moved? If this is merged, it will create duplicate files

**Comment by bilelmoussaoui** [comment]
It should be ready to be merged :+1:
